src/Presentation/Nop.Web/App_Data/Localization/defaultResources.nopres.xml

```diff
@@ -17231,6 +17231,15 @@
   </LocaleResource>
   <LocaleResource Name="Reviews.Fields.Rating.Bad">
     <Value>Bad</Value>
+  </LocaleResource>    
+  <LocaleResource Name="Reviews.Fields.Rating.NotGood">
+    <Value>Not good</Value>
+  </LocaleResource>
+  <LocaleResource Name="Reviews.Fields.Rating.NotBadNotExcellent">
+    <Value>Not bad but also not excellent</Value>
+  </LocaleResource>    
+  <LocaleResource Name="Reviews.Fields.Rating.Good">

```

Comment:
Please sort it alphabetically and also add new locales to upgrade.sql file (all new locales are at the top of the script)

Reply:
I have sorted them alphabetically and added them also to the upgrade.sql file, and committed the changes.